src/darwin/Framework/CHIP/MTRDevice.mm

```diff
@@ -610,6 +621,12 @@ - (void)_setupSubscription
                                // OnUnsolicitedMessageFromPublisher
                                [self _handleUnsolicitedMessageFromPublisher];
                            });
+                       },
+                       ^(void) {
+                           MTR_LOG_DEFAULT("%@ got report begin", self);
+                           dispatch_async(self.queue, ^{
+                               [self _handleReportBegin];
```

Comment:
So this will report reachable, but none of the attribute values will be reported, or even placed in the cache, until the subscription priming is done?

What I would have thought we would do is:

1. Mark as reachable.
2. Fill in the cache as we go, so reads during priming might get useful values.
3. Perhaps delay reports from MTRDevice to _its_ delegate, if we want to delay that sort of thing.

I suppose that can be a separate PR, but doing that would change the shape of things here a bit, because we would just update reachability when we get handed actual attribute data and not need the extra callback....

Reply:
I have an issue to surface reports as they come in. This will be done next:

https://github.com/project-chip/connectedhomeip/issues/26012